doc/release/launcher.rst

```diff
@@ -91,122 +94,309 @@ installation-specific configuration, e.g.), you can often use the ``-v``
 flag to capture the commands that the launcher executes on your behalf
 and customize them for your needs.
 
-
---------------------------------
 Forwarding Environment Variables
---------------------------------
+++++++++++++++++++++++++++++++++
 
 Chapel launchers generally arrange for environment variables to be
 forwarded to worker processes. However, this strategy is not always
 reliable. The remote system may override some environment variables, and
 some launchers might not correctly forward all environment variables.
 
 
----------------------------------
-_real binary suffix for execution
----------------------------------
+.. _using-udp:
 
-In order to support profiling tools that produce new binaries for the
-launcher to execute, the suffix of the real binary executed by the
-launcher may be changed with the ``CHPL_LAUNCHER_SUFFIX`` environment
-variable. If this variable is unset, the suffix defaults to "_real",
-matching the compiler's output.
+Using the Portable UDP Conduit
```

Comment:
I think having an idea of a "network" is reasonable in that future. We could view MPI and libfabric as networks and also UGNI if we cared to. I think it makes sense to keep UGNI documented in platforms/cray for now, but I could see moving it.

But, anyway, I think part of my view here is that our documentation needs to offer tested HOWTOs for common situations. If libfabric+ some particular hardware becomes somewhat common, we'll want to document that. If libfabric becomes the better way for Chapel to work with InfiniBand (say), I'd still want users to find infiniband.rst and just see information about libfabric there.

In other words - I think it's reasonable to expect our users know what networking hardware they're trying to use. I think the exceptions are MPI and UDP - I think our users will know when they want to use MPI, and UDP will work (slowly) most everywhere. Otherwise, I'd expect they'd follow whatever suggestion we offer (ie should I use MPI, GASNet verbs, GASNet mxm, libfabric, ...? with InfiniBand?).